package.json

```diff
@@ -25,7 +25,7 @@
   },
   "dependencies": {
     "chalk": "^2.1.0",
-    "cli-table2": "^0.2.0",
+    "cli-table3": "^0.5.0",
```

Comment:
I think you will need to use 0.4.0 or update `.travis.yml`/`engines`/etc to drop Node 4.

Reply:
I think dropping Node.js 4 is better as it reached its EOL.

Reply:
Totally agree, but it generally requires a major version bump so I figured I'd point out the option...

Reply:
I tend to agree. Let's us v0.4.0 for now, and then we can drop Node 4 in a separate step :)

Reply:
Changed it to 0.4.0.